platform/mbed_error.c

```diff
@@ -211,7 +211,9 @@ mbed_error_status_t mbed_error_initialize(void)
         if ((report_error_ctx->crc_error_ctx == crc_val) && (report_error_ctx->is_error_processed == 0)) {
             is_reboot_error_valid = true;
             //Report the error info
+#ifndef NDEBUG
```

Comment:
It would make sense for this to use `mbed_error_printf`, so you're definitely not pulling in anything extra beyond the existing error prints.

Reply:
👍 

These are the only 2 printf in this file,